recipes/costa/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "costa" %}
+{% set version = "2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/eth-cscs/costa/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 0692a9c962282899894ccdbc7006216c835473a6999f1080c10bf94f23094521
+
+build:
+  number: 0
+  skip: True  # [win]
+
+outputs:
+  - name: costa
+    build:
+      script_env:
+        - COSTA_SCALAPACK: OFF
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - cmake
+      host:
+        - {{ mpi }}
+        - libgomp      # [linux]
+        - llvm-openmp  # [osx]
+      run:
+        - {{ mpi }}
+    
+    test:
+      commands:
+        - test -f "${PREFIX}/lib64/libcosta.a"
+
+    about:
+      summary: COSTA
+
+  - name: costa-scalapack
+    build:
+      script_env:
+        - COSTA_SCALAPACK: CUSTOM
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - cmake
+      host:
+        - {{ mpi }}
+        - scalapack
+        - libgomp      # [linux]
+        - llvm-openmp  # [osx]
+      run:
+        - {{ mpi }}
+        - scalapack
+    
+    test:
+      commands:
+        - test -f "${PREFIX}/lib64/libcosta.a"
+        - test -f "${PREFIX}/lib64/libcosta_scalapack.a"
+        - test -f "${PREFIX}/lib64/libcosta_prefixed_scalapack.a"
+
+    about:
+      summary: COSTA with scalapack wrappers for drop-in replacement
+
+
+about:
+  home: https://github.com/eth-cscs/costa.git
```

Comment:
The repo name is originally capitalized, but it seems it still works :)

Reply:
thanks fixed

Reply:
great,  thanks!